src/interfaces/uproxy_core_api.ts

```diff
@@ -290,5 +302,12 @@ export interface CoreApi {
   // This may also invoke an OAuth flow, in order to perform operations
   // with the cloud computing provider on the user's behalf.
   cloudInstall(args:CloudInstallArgs): Promise<void>;
+
+  // Invokes OAuth flow and destroys uProxy cloud server on the 
+  // provider specified.
+  cloudDestroy(providerName :string) : Promise<void>;
```

Comment:
Hmm. We could define a bunch of finer-grained **args**, e.g. `CloudUpdateArgs` and `CloudDestroyArgs`, and have `cloudInstall` accept any one of those - as a union type. WDYT?